webapp/src/errorList.tsx

```diff
@@ -66,7 +64,7 @@ export class ErrorList extends React.Component<ErrorListProps, ErrorListState> {
                 errorListContent = (
                     <div className="ui selection list">
                         {(errors).map((e, index) =>
-                        <div className="item" key={errorKey(e)} role="button" onClick={createOnErrorMessageClick(e, index)}>
+                        <div className="item" key={errorKey(e)} role="button" onClick={createOnErrorMessageClick(e, index)} onKeyDown={sui.fireClickOnEnter} tabIndex={0}>
```

Comment:
Maybe have a label that specifies that this will "go to error ", but thats a bit of a nitpick